### PR TITLE
Add Readme and postman req/res examples.

### PR DESCRIPTION
**Pull request by jbguardian**
Adding ReadMe and POSTMAN examples.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,24 @@
...
+The payment service provides an API for creating PayPal and Stripe payments, consumed by Apps and Website. 
+
+### URL Healthcheack
+
+* Production: https://payment.guardianapis.com/healthcheck
+* CODE: https://payment.code.dev-guardianapis.com/healthcheck
+
+### Running Locally
+
+Get AWS credentials form Janus
```
Comment (joelochlann) [inline README.md]: typo: from

File: README.md [hunk 2]
```diff
@@ -1 +1,24 @@
-# Payment API
+# Payment Service
+
+The payment service provides an API for creating PayPal and Stripe payments, consumed by Apps and Website.
```
Comment (joelochlann) [inline README.md]: let's ditch the "consumed by" bit for the minute since it's subject to change

File: README.md (hunk 1 above)
Comment (jbguardian) [reply README.md]: 👍

File: README.md (hunk 2 above)
Comment (jbguardian) [reply README.md]: 👍

**Pull request merged**